packages/datetime/src/shortcuts.tsx

```diff
@@ -46,7 +46,7 @@ export class Shortcuts extends React.PureComponent<IShortcutsProps> {
 
         const shortcutElements = shortcuts.map((s, i) => (
             <MenuItem
-                className={Classes.POPOVER_DISMISS_OVERRIDE}
+                shouldDismissPopover={false}
```

Comment:
mm it's not entirely this simple. can we actually make this value correspond to the `closeOnSelection` prop?
so if you enable that prop then clicking shortcuts will close, otherwise they won't.